mu4/appshell/appshell.cpp

```diff
@@ -55,10 +55,10 @@ int AppShell::run(int argc, char** argv, std::function<void()> moduleSetup)
 
     qSetMessagePattern("%{function}: %{message}");
 
-    framework::settings()->load();
-
     moduleSetup();
 
+    framework::settings()->load();
```

Comment:
What is the motivation of the transfer?

Reply:
Previously, we first loaded the values(there were no registered keys, so there was nothing to load), then we registered keys. It's just a bug